.github/workflows/unit-tests.yml

```diff
@@ -31,6 +31,8 @@ jobs:
 
       - name: Install dependencies
         run: |
+          sudo apt update -y
+          sudo apt install libgirepository1.0-dev gcc libcairo2-dev pkg-config python3-dev gir1.2-gtk-4.0 -y
```

Comment:
We could create our own container for running the tests with these dependencies already installed to avoid having to run it every time.

Reply:
Agreed. But I think I will let the container for another time. I will crete a ticket for it